src/modules/variables.js

```diff
@@ -0,0 +1,4 @@
+export const form = document.getElementById('todoform');
+export const todoInput = document.getElementById('newtodo');
+export const todosListEl = document.getElementById('todos-list');
+export const notificationEl = document.querySelector('.notification');
```

Comment:
As mentioned I Suggest you create a class for each html node that has the nodes functionality i.e color changes, event handlers etc because this is part of the best coding principles and it makes other people who will be reading/using your code easily understand it😉

Reply:
[Issue resolved.](https://github.com/Lucash2022/To-Do-list-review/pull/1/commits/781b03ea9c0bfe433be0380d407f67a066d9a8bf) Thank you